### PR TITLE
German pages: replace `Mehr Informationen` with `Weitere Informationen`

### PR DESCRIPTION
**Pull request by marchersimon**
This is something I wanted to do for quite a while. "Mehr Informationen" as a translation for "More information" is fine, but "Weitere Informationen" feels a bit more natural and professional.

### CODE REVIEW

File: scripts/set-more-info-link.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     'en': 'More information:',
     'bs': 'Više informacija:',
     'da': 'Mere information:',
-    'de': 'Mehr Informationen:',
+    'de': 'Weitere Informationen:',
```
Comment (bl-ue) [inline scripts/set-more-info-link.py:11]: Good job for remembering 😅

**Review by bl-ue — APPROVED** [review]

**Comment by marchersimon** [comment]
> Interesting. thinking
> 
> For reference, here's where I decided to use Mehr: #5140

I asked a few friends of mine and they thought it sounds nicer. However that might differ from area to area. I think I'll ask in some German forum for a definitive answer.

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

**Comment by marchersimon** [comment]
I'll convert it to a draft for now to avoid merging until I have a definitive answer.

**Comment by FantasyCookie17** [comment]
I'd say that they both sound equally fine, and "mehr" is shorter…

**Comment by marchersimon** [comment]
My findings so far:
* According to [Google Trends](https://trends.google.com/trends/explore?geo=DE&q=mehr%20Informationen,weitere%20Informationen), `Weitere Informationen` seems to be more popular.
* On [r/German](https://www.reddit.com/r/German/comments/mwr0hh/mehr_informationen_or_weitere_informationen/), I got two interesting answers:

> "Mehr" alludes to missing information. So if you need to know more you need mehr Informationen. If you have already provided sufficient information but want to encourage learning more you can link to weitere Informationen.

> To me, "mehr" implies quantity, and "further" implies more-in-depth.

**Comment by bl-ue** [comment]
That sounds great.  I think we have our answer.  I'll merge now, given that it appears to be done.

**Pull request merged**